카카오 로그인

yooxdb opened the pull request:
- 로컬에서 갑자기 이상한 파일들이 추가되어 확인해보려고 PR 먼저 올립니다
- 카카오 로그인 기능은 구현된 상태이고, 다듬는 중입니다

yooxdb committed:
Add KakaoLoginView + Controller

yooxdb committed:
Add KakaoAuthManager

yooxdb committed:
Add Setting for using KakaoSDK

yooxdb committed:
Add Endpoint+Kakao

yooxdb committed:
Add Kakao Connect + Login feature

yooxdb committed:
Merge branch 'master' into feat/kakao-login

yooxdb committed:
Refactor Observable -> Single (일부)

- 로그인 및 회원가입 네트워크 요청이 2번 호출되는 문제 해결

yooxdb committed:
Refactor AuthManager to class

yooxdb committed:
Add KakaoLoginButton

yooxdb committed:
Edit Username Textfield Placeholder

shp7724 reviewed (approved):
미리 Approve해두겠습니다.
화이팅입니다.

yooxdb committed:
Add KakaoLoginView

yooxdb committed:
Fix 사용자의 취소 or 에러 구분

- 카카오 계정을 connect할 때 단순히 유저가 cancel을 탭해서 취소하는 경우와 connect를 시도하였으나 에러가 발생한 경우를 구분

yooxdb committed:
Add WafflebookLogo Image

yooxdb committed:
Fix f9ea57 again

yooxdb committed:
Remove Grayscale .gray3

- to resolve conflict
